api/v1alpha4/virtualmachine_storage_types.go

```diff
@@ -202,14 +202,14 @@ type VirtualMachineStorageStatusUsed struct {
 	// +optional
 
 	// Disks describes the total storage space used by a VirtualMachine's
-	// non-PVC disks.
+	// non-PVC disks. This does not include the snapshot related disks.
 	Disks *resource.Quantity `json:"disks,omitempty"`
 
 	// +optional
 
 	// Other describes the total storage space used by the VirtualMachine's
-	// non disk files, ex. the configuration file, swap space, logs, snapshots,
-	// etc.
+	// non disk files, ex. the configuration file, swap space, logs, etc.
+	// This does not include the snapshot related non disk files.
```

Comment:
```suggestion
	// This does not include the any child / delta disks created for snapshots.
    ```

Same in other places maybe?